osquery/sql/sqlite_encoding.cpp

```diff
@@ -16,10 +16,16 @@
 
 namespace osquery {
 
+enum B64Type {
+  B64_ENCODE_CONDITIONAL = 1,
+  B64_ENCODE = 2,
+  B64_DECODE = 4,
```

Comment:
Can you give some examples of how these `enum` values are toggled? Having 1, 2, and 4 used to toggle these values seems silly. I get the binary fluency and logical operators, but if this is something the user is directly setting it'd be better to just have 1, 2, and 3, but as I said I'm likely misunderstanding the usage here. Relating to the point below, would you mind giving a sample of the usage of `B64_ENCODE_CONDITIONAL` as well as `B64_ENCODE`?